tools/diagnostics/create_service_diagnostics_bundle.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+readonly DCOS_COMMONS_SCRIPT_PATH="tools/diagnostics"
+
+function is_dcos_commons_repository_script() {
+  [[ ${SCRIPT_DIRECTORY} = *${DCOS_COMMONS_SCRIPT_PATH} ]]
+}
+
+function is_development_mode() {
+  is_dcos_commons_repository_script
+}
+
+if is_development_mode; then
+  echo "dcos-commons repository detected, running in development mode"
+  echo
+  echo "In development mode all Python modules will be picked up from your current"
+  echo "dcos-commons repository instead of the Docker image's /dcos-commons-dist "
+  echo "directory that contains a static git checkout"
+  echo
+  set -x
+  readonly CONTAINER_DCOS_COMMONS_DIRECTORY="/dcos-commons"
+  readonly CONTAINER_DCOS_COMMONS_VOLUME_MOUNT="-v $(pwd):${CONTAINER_DCOS_COMMONS_DIRECTORY}:ro"
+else
+  readonly CONTAINER_DCOS_COMMONS_DIRECTORY="/dcos-commons-dist"
+  # We don't mount the dcos-commons directory in the container because the
+  # script will use /dcos-commons-dist which is added to the Docker image during
+  # build time.
+  readonly CONTAINER_DCOS_COMMONS_VOLUME_MOUNT=
+fi
+
+readonly VERSION='v0.1.0'
+
+function version () {
+  echo "${VERSION}"
+}
+
+function usage () {
+  echo 'Usage: ./create_service_diagnostics_bundle.sh <package name> <service name> [<proceed>]'
```

Comment:
Would it make sense to just pass the arguments directly to the python script -- then the command line will be the same? This will also allow proceed to work as expected.

Reply:
Good idea: d47214d860e09f212ffd3c0f8d6d9a61aece4fef